.gitignore

```diff
@@ -4,6 +4,8 @@
 # or operating system, you probably want to add a global ignore instead:
 #   git config --global core.excludesfile '~/.gitignore_global'
 
+node_modules
+
```

Comment:
- [ ] Good job so far 👏 It is really great that you have added `node_modules` to this `.gitignore` file. However, I can say that this is the first step to remove the node_modules folder from GitHub because since you have already pushed it is still there. I am going to give you suggestions about how you can remove it from Github too.
 First way, in a case like that, you can untrack the `node_modules` folder by 
```shell
git rm -r --cached path_to_the_folder
```
and then, go and delete it on Github just manually 👍 
Second way, 
1. is to delete it locally in your repo,

2.Create a new commit with the changes. You can do this by running the command `git add .` to stage all changes, followed by `git commit -m "Removed node_modules folder"` to create a new commit with a message explaining the change.

3. Push the changes to GitHub. You can do this by running the command git push, which will update the remote repository with the new commit.

But as you have already pushed the node_modules folder to GitHub, you will need to force push the changes to overwrite the remote repository. You can do this by running the command `git push -f`, which will force push the new commit to GitHub.

I hope you will find this useful 😃